client/public/static/js/recipes/recipes.js

```diff
@@ -16,14 +20,22 @@ function RecipesView() {
     try {
       // Show publish recipe button
       const hidePublicRecipeButton = document.getElementById('togglePublicRecipeShownButton');
+      hidePublicRecipeButton.textContent = showPublicRecipes ? HIDE_PUBLIC_RECIPES : SHOW_PUBLIC_RECIPES;
       if(showPublicRecipes){
```

Comment:
These next five lines are now redundant as you made this nicer ternary operation